app/src/components/annotation/AnnotationTooltip.tsx

```diff
@@ -45,12 +45,14 @@ export function AnnotationTooltip({
             </Text>
             <Text color="inherit">{annotation.label || "--"}</Text>
           </Flex>
-          <Flex direction="row" justifyContent="space-between">
-            <Text weight="heavy" color="inherit">
-              kind
-            </Text>
-            <Text color="inherit">{annotation.annotatorKind}</Text>
-          </Flex>
+          {annotation.annotatorKind ? (
+            <Flex direction="row" justifyContent="space-between">
+              <Text weight="heavy" color="inherit">
+                kind
```

Comment:
can we spell out annotator? I think otherwise it's a bit confusing. That or I might go as far as saying let's hide this for now.

Reply:
i added annotator - think it makes sense to leave in but can remove if folks find it confusing. I myself found it useful for looking at which things i had added vs pre-existing evals